### PR TITLE
Multilingual site title, description and meta tags

### PR DESCRIPTION
**Pull request by nmpetkov**
With this PR I'm trying to fix a long awaiting multilingual issue.

| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | yes
| BC breaks?        | no
| Deprecations?     | no
| Tests pass?       | yes
| Fixed tickets     | #777, #519
| Refs tickets      | -
| License           | MIT
| Doc PR            | -
| Changelog updated | yes

Some pictures from the interface for entering multilingual items. This is of course subject to change, if sombody like to make better interface.

image

image

Please check changed files, to discuss if this is the best way to implement.

### CODE REVIEW

File: src/lib/util/ModUtil.php [hunk 1]
```diff
@@ -138,6 +138,23 @@ public static function initCoreVars($force=false)
...
 
     /**
+     * Init variables multilingual dependent.
+     *
+     * @return boolean True
+     */
+    public static function setupMultilingual()
+    {
+        $lang = ZLanguage::getLanguageCode();
+        LogUtil::registerStatus('ModUtil: '.$lang);
```
Comment (craigh) [inline src/lib/util/ModUtil.php]: purpose of this line?

File: src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php [hunk 2]
```diff
@@ -30,7 +30,7 @@ public function getMetaData()
         $meta['description'] = $this->__('General site configuration interface.');
         //! module name that appears in URL
         $meta['url'] = $this->__('settings');
-        $meta['version'] = '2.9.9';
+        $meta['version'] = '3.0.0';
```
Comment (craigh) [inline src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php]: probably `2.9.10` would be more appropriate. changing the first digit implies much larger changes.

File: src/lib/util/ModUtil.php (hunk 1 above)
Comment (nmpetkov) [inline src/lib/util/ModUtil.php]: Sorry, this is from test, I'l remove now!

**Comment by craigh** [comment]
what happens if multilang entires are left blank?

**Comment by craigh** [comment]
this will require a changelog entry please. update the header template with the ticket and changelog update status.

File: src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php (hunk 2 above)
Comment (nmpetkov) [inline src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php]: I attempted, but the way switch works prevent to work...

Comment (craigh) [inline src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php]: I don't understand. please explain more

Comment (nmpetkov) [inline src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php]: If we preffer 2.9.10, then have to implement other logic.

**Comment by nmpetkov** [comment]
If the are blank - just this setting is also blank.

File: src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php (hunk 2 above)
Comment (nmpetkov) [inline src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php]: I'l check this again.

**Comment by nmpetkov** [comment]
Edited QA table and added entry in change log.

**Comment by craigh** [comment]
ping @Guite please merge if you support.

**Pull request merged**